src/sagemaker_sklearn_extension/preprocessing/encoders.py

```diff
@@ -361,6 +370,19 @@ def inverse_transform(self, y):
         y_decoded = [self.classes_[idx] if idx in labels else self.fill_label_value for idx in y]
         return y_decoded
 
+    def get_classes(self):
+        """Returns the values of the unencoded classes.
+        If ``self.include_unseen_class`` is ``True`` include ``self.fill_label_value`` as a class.
+
+        Returns
+        -------
+        classes : array of shape (n_classes,)
+        """
+        if not self.include_unseen_class or not self.fill_unseen_labels:
+            return self.classes_
+
+        return self.classes_ + [self.fill_label_value]
```

Comment:
`self.classes_` is a numpy array of dtype string. I think if you try to add a fill_label_value it'll cause an error? This is what i got after a quick test
```
>>> rle.classes_
array(['a', 'b', 'e'], dtype='<U1')
>>> rle.classes_ + ['c']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
numpy.core._exceptions.UFuncTypeError: ufunc 'add' did not contain a loop with signature matching types (dtype('<U1'), dtype('<U1')) -> dtype('<U1')
```

Reply:
Also might be a problem if dtype is different as well?

Reply:
This succeeds on my computer, but I will make an update that uses `np.append` to be sure this change is functional.